### ISSUE TITLE
Organizing regional Hydra meetings

### ISSUE THREAD

**Issue by lsat12357** [projecthydra-labs/hydra-connect#16]
Organizing regional Hydra meetings

Suggested by: Chrissy Rissmeyer
Presenter: Organizers of previous regional meetings (some possibilities: Linda Newman, Karen Cariani, Chrissy Rissmeyer, Matt Critchlow, Arwen Hutt, others?)
Format: Panel
Audience: All

**Comment by richardGreen** [comment]
Panel - try and get it into the parallels.  Need confirmed names.

**Comment by richardGreen** [comment]
2016-08-09  So far, myself (representing the West), Karen Cariani (representing NE/the community guidelines), and Linda Newman (representing the community guidelines) have confirmed their willingness to participate. Still waiting to hear back from some others, but hopefully confirmation from the three of us is enough for the planning committee to move forward at this point.
- Chris Awre or Julie Allinson

**Comment by richardGreen** [comment]
NEED & SYNOPSIS

**Comment by richardGreen** [comment]
2016-08-15
Regional meetings are a great way to stay connected with your fellow Hydranauts throughout the year. Hear from past organizers about their experiences and tips, and learn about community resources to help get you started. It is also a chance to connect with other individuals in your region who are interested in organizing a future event.

**Comment by richardGreen** [comment]
8/18
That is great news! I have been able to confirm the following individuals for the panel:
• Julie Allinson (University of York)
• Karen Cariani (WGBH)
• Linda Newman (University of Cincinnati)
• Joseph Rhoads (Brown University)
• Chrissy Rissmeyer (UC Santa Barbara)
Arwen and Matt will not be participating on the panel. Matt Critchlow and his wife just welcomed their first child (a little boy), so he will not be attending this year's Connect. I spoke with Arwen yesterday, and she has decided to sit this one out since we were able to find panelists from other regions to participate. She does plan to attend the session, however.

I'll touch base with the other panelists about their willingness to be recorded and the panel description on the wiki.
